e2e-tests/cypress/integration/appeal-submission-tasklist.feature

```diff
@@ -0,0 +1,22 @@
+@wip
+Feature: Your Details status
+
+    Scenario Outline: The status of the Your Details section is accurate
+        # correct this, and correct your-details feature; 'valid' should not feature here; we want to be able to re-use for invalid cases too
+        When the user provides their valid <name> and <email>
+        # correct this. update who=are=you feature to accept "yes"/"no" or "is"/"isnt"
+        And the user states that they <original appelant> the original appellant
+        # correct this. update who-are-you feature to care about the original apelant name - this sentence should appear in that feature somewhere?
+        And the user provides <on behalf of> as the name of the original appellant
+        Then the user should see that Your Details section is <status>
+        
+        Examples:
+            | name                | email            | original apellant | on behalf of    | Status        |
+            | ""                  | ""               | "doesnt matter"   | "doesnt matter" | "Not Started" |
+            | ""                  | "good@email.com" | "doesnt matter"   | "doesnt matter" | "Not Started" |
```

Comment:
Will it be possible to get into this state where there is no name but there is an email? I think if this is submitted it should fail validation and prevent any update.

Reply:
that's the situation we're trying to communicate here - so if we give no name + an email we should fail validation so the status should be 'not started'.
totally aware there may be nicer ways to word things but i was trying to show that in a situation where we've given something invalid:  it doesn't matter what we do for the other fields (maybe "" is clearer there?)

Reply:
Ok.